plugin/docker/container-metadata.go

```diff
@@ -13,16 +13,36 @@ type containerMetadata struct {
 	container *container
 }
 
+func (cm *containerMetadata) Metadata(ctx context.Context) (plugin.EntryMetadata, error) {
```

Comment:
This seems not ok. `CachedMetadata` -> `Metadata` -> `content` -> `CachedMetadata`. Either `Metadata` should call `CachedOpen` or `Open` should call `CachedMetadata`.

Reply:
I think it should be b/c it calls container's CachedMetadata. However, these should use `Open` -- that makes a lot more sense. Will update.

Reply:
Updated.